smac/smac_cli.py

```diff
@@ -31,21 +31,33 @@ def __init__(self):
         '''
             constructor
         '''
-        self.logger = logging.getLogger(self.__module__ + "." + self.__class__.__name__)
+        self.logger = logging.getLogger(
+            self.__module__ + "." + self.__class__.__name__)
 
     def main_cli(self):
         '''
             main function of SMAC for CLI interface
         '''
-        self.logger.info("SMAC call: %s" %(" ".join(sys.argv)))
+        self.logger.info("SMAC call: %s" % (" ".join(sys.argv)))
 
         cmd_reader = CMDReader()
         args_, misc_args = cmd_reader.read_cmd()
 
-        logging.basicConfig(level=args_.verbose_level)
-
         root_logger = logging.getLogger()
         root_logger.setLevel(args_.verbose_level)
+        logger_handler = logging.StreamHandler(
+                stream=sys.stdout)
+        if root_logger.level >= 20:
+            formatter = logging.Formatter(
+                "%(levelname)s:\t%(message)s")
```

Comment:
I understand that you don't want to have the name of the logger here, but we could also add the time in H:M:S format here. I think this would be highly useful.